History.md

```diff
@@ -259,6 +297,24 @@
 
 ## v1.7.0.1, 2018-05-29
 
+### Breaking changes
+* You may need to change your aggregations calls. Now `Meteor.wrapAsync(rawCollection.aggreate)` will return an `‌AggregationCursor` and no longer the aggregation result directly, you can fix this changing your code to:
```

Comment:
Would it be possible to wrap these lines at 80 characters like the rest of the file?